Remove English from MR interface files (new)

thesunshade opened the pull request.

thesunshade committed:
Remove English from MR interface files

ccronje commented:
Hi
@ihongda there's a comma in `share_translation-mr-site.json`
@thesunshade can you please add a step before pull request to test in a JSON validator (e.g. https://jsonlint.com/)
Thanks!

thesunshade commented:
Again, very sorry about this.

I have fixed with https://github.com/suttacentral/bilara-data/pull/3276

I think after we split the interface file into several smaller files I must have gotten lazy. VS code (for me) has eslint checking everything so I was relying on that. However this time I made them in Notepad++, which has a linter for json, but for some reason it is fine with that comma after the last item.
